src/test/suite/fixtures/recorded/navigationMap/moveFourthCarHarpPastSecondCarWhaleToEndOfWhaleTakeWhale.yml

```diff
@@ -0,0 +1,54 @@
+spokenForm: move fourth car harp past second car whale to end of whale take whale
+languageId: plaintext
+command:
+  actionName: moveToTarget
+  partialTargets:
+    - type: range
+      start:
+        type: primitive
+        selectionType: token
+        modifier: {type: subpiece, pieceType: character, anchor: 3, active: 3, excludeAnchor: false, excludeActive: false}
+        mark: {type: decoratedSymbol, symbolColor: default, character: h}
+      end:
+        type: primitive
+        selectionType: token
+        modifier: {type: subpiece, pieceType: character, anchor: 1, active: 1, excludeAnchor: false, excludeActive: false}
+        mark: {type: decoratedSymbol, symbolColor: default, character: w}
+      excludeStart: false
+      excludeEnd: false
+    - type: primitive
+      position: after
+      insideOutsideType: inside
+      mark: {type: decoratedSymbol, symbolColor: default, character: w}
+  extraArgs: []
+initialState:
+  documentContents: hello world whatever
+  selections:
+    - anchor: {line: 0, character: 20}
+      active: {line: 0, character: 20}
+  marks:
+    default.h:
+      start: {line: 0, character: 0}
+      end: {line: 0, character: 5}
+    default.w:
+      start: {line: 0, character: 6}
+      end: {line: 0, character: 11}
+finalState:
+  documentContents: helrldlo wo whatever
+  selections:
+    - anchor: {line: 0, character: 20}
+      active: {line: 0, character: 20}
+  marks:
+    default.h:
+      start: {line: 0, character: 0}
+      end: {line: 0, character: 3}
+    default.w:
+      start: {line: 0, character: 3}
+      end: {line: 0, character: 8}
```

Comment:
This test is kinda bananas, but basically we've deleted first 2 characters of whale, then added stuff with a space in it to the end of whale.  We preserve those last 3 chars of whale and pick up the test that was added until the space on the end! 😅